code/modules/mapping/merge_conflicts.dm

```diff
@@ -0,0 +1,17 @@
+// Used by mapmerge2 to denote the existence of a merge conflict (or when it has to complete a "best intent" merge where it dumps the movable contents of an old key and a new key on the same tile).
+// We define it explicitly here to ensure that it shows up on the highest possible plane (while giving off a verbose icon) to aide mappers in resolving these conflicts.
+// DO NOT USE THIS IN NORMAL MAPPING!!! Linters WILL fail.
+
+/obj/merge_conflict_marker
+	name = "Merge Conflict Marker - DO NOT USE"
+	icon = 'icons/effects/mapping_helpers.dmi'
+	icon_state = "merge_conflict_marker"
+	desc = "If you are seeing this in-game: someone REALLY, REALLY, REALLY fucked up. They physically mapped in a fucking Merge Conflict Marker. What the shit."
+	plane = POINT_PLANE
+
+///We REALLY do not want un-addressed merge conflicts in maps for an inexhaustible list of reasons. This should help ensure that this will not be missed in case linters fail to catch it for any reason what-so-ever.
+/obj/merge_conflict_marker/Initialize(mapload)
+	. = ..()
+	var/msg = "HEY, LISTEN!!! Merge Conflict Marker detected at [AREACOORD(src)]! Please manually address all potential merge conflicts!!!"
+	warning(msg)
+	to_chat(world, "<span class='boldannounce'>[msg]</span>")
```

Comment:
I don't know how I feel about this, if there are 100+ conflicts, this has the potential to start really lagging/crashing clients based on chatload. Perhaps it would be better to create a large debug log at the end of initialization indicating the existence of merge_conflict_markers and then just being able to click on a button that opens up a browserUI menu listing each coord?

Reply:
There should NEVER be a merge conflic marker on prod. If it does the round is likely screwed massively anyway as a map failed to merge properly.

Reply:
If these throw a stack trace or exception they will increment world runtime count which will fail CI, so they should never get in anyway.

Reply:
I know this will never make it onto production, I'm just saying that when people load this up for testing

Reply:
I'm comfortable with collapsing all the errors into a global counter of found markers but I'm not sure where I'd put the log invocation post-initialize. Would that work for everyone?

Reply:
No strong consensus on this so I'm going to keep it as is, and if we encounter issues when testing, I'll streamline the error collection here.